pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -60,8 +60,8 @@ stdenvNoCC.mkDerivation rec {
   outputHashAlgo = "sha256";
   outputHashMode = "recursive";
   outputHash = if stdenvNoCC.isDarwin
-    then "00d49ls9vcjan1ngq2wx2q4p6lnm05zwh67hsmj7bnq43ykrfibw"
-    else "1amagcaan0hk3x9v7gg03gkw02n066v4kmjb32yyzsy5rfrivb1a";
+    then "00d49ls9vcjan1ngq2wx2q4p6lnm05zwh67hsmj7bnq44ykrfibw"
```

Comment:
Last time I did this I changed the hash to all 0's to make it very clear that the Darwin one is busted until someone with a Mac computes it, which was done here: https://github.com/NixOS/nixpkgs/pull/52950

That said, it looks like @smaret is actively using the Mac version (https://github.com/NixOS/nixpkgs/pull/57947) and might be able to compute the Mac hash for this PR.

Reply:
```suggestion
    then "0rwm46v9amq2clm6wxhr98zzbafr485dz05pihlqsbrbabmlfw30"
```

Reply:
thanks!